tests/manifest.ttl

```diff
@@ -1,17 +1,27 @@
-## Distributed under both the "W3C Test Suite License" [1]
-## and the "W3C 3-clause BSD License".
 ## [1] https://www.w3.org/Consortium/Legal/2008/04-testsuite-license
 ## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
 
-PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
-PREFIX rdft:   <http://www.w3.org/ns/rdftest#>
-PREFIX trs:    <https://w3c.github.io/rdf-star/tests#>
+@PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@PREFIX rdft:   <http://www.w3.org/ns/rdftest#> .
+@PREFIX trs:    <https://w3c.github.io/rdf-star/tests#> .
+@PREFIX dct: <http://purl.org/dc/terms/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
 
 trs:manifest  rdf:type mf:Manifest ;
-    rdfs:label "RDF-star test suite" ;
-    rdfs:seeAlso <https://w3c.github.io/rdf-tests/> ;
+  rdfs:label "RDF-star test suite"@en ;
+	rdfs:label "La suite des tests pour RDF-star"@fr;
+	rdfs:label "Conjuto de pruebas para RDF-star"@es;
+	dct:issued "2021-06-21"^^xsd:date ; #not sure if this was the first release date
+	dct:creator [ foaf:name "Gregg Kellog" ] ; #maybe add foaf:homepage
+	dct:creator [ foaf:name "Andy Seaborne" ];
+  rdfs:seeAlso <https://w3c.github.io/rdf-tests/> ;
+	dct:modified "2021-07-18"^^xsd:date ; # change to the exact date
```

Comment:
For the community to discuss: 

- dates get out of date (they are more maintenance work)
- the same date may not apply to all tests.

https://www.w3.org/ns/rdftest# has some per-test vocabulary.

Reply:
Per-test metadata will never be managed properly. It's more useful to maintain the modification date of the manifest, itself, which could be useful to ensure that EARL reports match the latest version of the test manifest. IIRC, @pchampin suggested something like this elsewhere.

Reply:
As long as modified !=> created in any way (another reason to remove dct:creator -- there isn't a neat "create" point for a manifest).

The https://www.w3.org/ns/rdftest#  vocabulary has been used for per test when the WG formally accepted tests. I agree that in the community space, maintenance is burden.

Reply:
Per-test metadata *should* be manageable, and I think setting that example may be a subtle goodness we can bring to the general community.  `dct:creator`, `dct:editor`, `dct:modified`, `dct:created` are all reasonable.  I have certainly had times where such info would have been valuable to me, about RDF artifacts as well as HTML, TXT, PDF, etc.

Reply:
It's easy enough to support additional properties on a test entry, but the code that transforms to JSON-LD and HTML will require some minor updates to make sure this doesn't get lost. Presumably, much metadata could be discovered through GitHub commit history, and automating this might be the best way to go, at least to get existing tests seeded with data.